iaso/models/project.py

```diff
@@ -58,7 +58,13 @@ class Project(models.Model):
     name = models.TextField(null=False, blank=False)
     forms = models.ManyToManyField("Form", blank=True, related_name="projects")
     account = models.ForeignKey("Account", on_delete=models.DO_NOTHING, null=True, blank=True)
-    app_id = models.TextField(null=True, blank=True)
+    app_id = models.TextField(
+        blank=True,
+        # Empty values are stored as NULL if both `null=True` and `unique=True` are set.
+        # This avoids unique constraint violations when saving multiple objects with blank values.
+        null=True,
+        unique=True,
+    )
```

Comment:
@madewulf Is it normal that `app_id` is nullable in `Project`? Shouldn't we fix that?